libs/sdk-ui-kit/src/SettingItem/tests/SettingItem.test.tsx

```diff
@@ -0,0 +1,59 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import { render, screen, waitFor } from "@testing-library/react";
+// import userEvent from "@testing-library/user-event";
+import { withIntl } from "@gooddata/sdk-ui";
+
+import { SettingItem } from "../SettingItem";
+import { ISettingItem } from "../typings";
+import userEvent from "@testing-library/user-event";
+
+const Wrapped = withIntl(SettingItem);
+
+describe("ReactSettingItem", () => {
+    it("should render setting item", () => {
+        const props: ISettingItem = {
+            title: "Setting Item title",
+            value: "Setting Item value",
+            actionType: "LinkButton",
+            actionValue: "Change",
+        };
+        const { container } = render(<Wrapped {...props} />);
+
+        expect(screen.getByText(props.title)).toBeInTheDocument();
+        expect(screen.getByText(props.value as string)).toBeInTheDocument();
+        expect(screen.getByText(props.actionValue as string)).toBeInTheDocument();
+        expect(container.getElementsByClassName("gd-button-link-dimmed").length).toBe(1);
+    });
+
+    it("should render setting item with Action is Switcher", () => {
```

Comment:
Please update the description a little bit to make it sound more naturally

```
"should render setting item with Action is Switcher"
```

↓

```
"should render setting item with Switcher as an action"
```

Reply:
fixed